float32 pm10
float32 pm25
float32 pm50
float32 pm100
float32 tmp
float32 hum
float32 co2